cmd/podman/containers/cp.go

```diff
@@ -93,6 +95,12 @@ func cp(cmd *cobra.Command, args []string) error {
 		return copyFromContainer(sourceContainerStr, sourcePath, destPath)
 	}
 
+	if followLink {
```

Comment:
Does it also apply to symlink _inside_ a container?

Reply:
I don't know.